esp-wifi/src/wifi/mod.rs

```diff
@@ -222,7 +222,9 @@ pub enum SecondaryChannel {
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
 pub struct AccessPointInfo {
     /// The SSID of the access point.
-    pub ssid: heapless::String<32>,
+    // TODO: we can use the `alloc` feature once we have `defmt` 1.0.2
+    #[cfg_attr(feature = "defmt", defmt(Debug2Format))]
+    pub ssid: alloc::string::String,
```

Comment:
`alloc::string::String` is present 14 times in this file, probably we can just import the type and avoid all this extra noise? 😁

Reply:
now that `heapless` is gone it's unambiguous and it's fine to do